### DISCUSSION TITLE
Forge stopped working with LORAs

### DISCUSSION THREAD

**Discussion by dudedudester4vr** [lllyasviel/stable-diffusion-webui-forge#2104] [Q&A]
I have been running Forge just fine and then all of a sudden it stopped working if I tried to use any LORA with it. I tried all the usual stuff i.e. fresh install, updating driver, restarting, updating python, updating CUDA etc. It still works without LORAs but when I try to run it with a LORA it makes my PC freeze.

**Comment by abzaloff** [comment]
choose here:
![Снимок экрана 2024-10-18 153341](https://github.com/user-attachments/assets/489a3d4e-c704-4ee0-a971-c768fc5e0e5d)

**Reply by dudedudester4vr** [reply]
thank you for the suggestion. Tried it, but unfortunately does not work.

**Reply by sw0ad** [reply]
in webui add "git reset --soft 210af4f80406f78a67e1c35a64a6febdf1200a82" (after @echo off) you will roll back update, this is the only thing that helped me

**Reply by mistermcluvin** [reply]
Interesting. I update before starting Forge daily so I am on the latest code and I have never had this issue with Loras crashing my PC(Win11). Your PC crashes when using any/all Loras or just Flux Loras? Do you recall if you installed another app around the same time you started having this issue? You are doing manual re-installs, not some pinokio type deal? Are you redirecting your Lora directory to another drive/directory?

**Reply by sw0ad** [reply]
in my case it started specifically after the update. I have 8 GB of VRAM, maybe that's the reason

**Reply by mistermcluvin** [reply]
Ahh, yes, maybe this is a Flux+PC hardware spec issue? I am on an old 5900x AM4 platform but I have 64GB RAM and an Asus 4090 w/ 24GB VRAM.

**Reply by sw0ad** [reply]
maybe, but I didn't use flux, only PDXL :)

**Reply by dudedudester4vr** [reply]
I don't use flux, takes too long to gen on my 3070. Yes it has 8gb vram, but it has been working fine for over a year now. There is no reason it should suddenly stop working.

I 

> Interesting. I update before starting Forge daily so I am on the latest code and I have never had this issue with Loras crashing my PC(Win11). Your PC crashes when using any/all Loras or just Flux Loras? Do you recall if you installed another app around the same time you started having this issue? You are doing manual re-installs, not some pinokio type deal? Are you redirecting your Lora directory to another drive/directory?

No I haven't installed anything specific. Some games maybe, but not even at same time as the problem started. 

> in webui add "git reset --soft [210af4f](https://github.com/lllyasviel/stable-diffusion-webui-forge/commit/210af4f80406f78a67e1c35a64a6febdf1200a82)" (after @echo off) you will roll back update, this is the only thing that helped me

The problem began all of a sudden. Not after an update.
